docs/source/CartesianConfig.rst

```diff
@@ -427,6 +345,169 @@ values. If in doubt of where to define or set a key, placing it at the
 top indentation level, at the end of the file, will guarantee it is
 used.
 
+Advanced features
+=================
+
+Variant sets combinations short names
+-------------------------------------
```

Comment:
It took me a while to understand this section name. How about `Short names for variant sets`? Or simply `Short names`? Or `Name vs. short_name`...